templates/demo-components/dynamic-components.php

```diff
@@ -0,0 +1,519 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Dynamic Components</title>
+</head>
+<body>
+	
+<?php
+
+use Tutor\Components\Accordion;
+use Tutor\Components\Avatar;
+use Tutor\Components\Badge;
+use Tutor\Components\Button;
+use Tutor\Components\InputField;
+use Tutor\Components\Modal;
+use Tutor\Components\Progress;
+use Tutor\Components\Tabs;
+
+?>
+<!-- button component  -->
+<div class="btn-wrapper tutor-mb-12">
+	<h2>Button</h2>
+	<pre><code>
+	&lt;?php
+		echo Button::make()->label( 'I am a button' )->size( 'large' )->variant( 'primary' )->render(); // phpcs:ignore
+		echo Button::make()->label( 'I am a button' )->size( 'medium' )->variant( 'primary' )->render(); // phpcs:ignore
+		echo Button::make()->label( 'I am a button' )->size( 'small' )->variant( 'primary' )->render(); // phpcs:ignore
+		echo Button::make()->label( 'I am a button' )->variant( 'primary-soft' )->render(); // phpcs:ignore 
+		echo Button::make()->label( 'I am a button' )->variant( 'destructive' )->render(); // phpcs:ignore 
+		echo Button::make()->label( 'I am a button' )->variant( 'destructive-soft' )->render(); // phpcs:ignore 
+	?&gt;
+	</code></pre>
+
+
+	<div class="tutor-dynamic-btn-wrapper tutor-flex tutor-gap-3 tutor-items-center tutor-flex-wrap">
+		<?php
+        echo Button::make()->label( 'I am a button' )->size( 'large' )->variant( 'primary' )->render(); // phpcs:ignore 
```

Comment:
The constants classes will help lot a developer in class usages. Easily predict the sizes and variants.

```php
echo Button::make()->label( 'I am a button' )->size( Size::LARGE )->variant( Variant::PRIMARY )->render();
```